app/src/lib/open-shell.ts

```diff
@@ -15,14 +15,20 @@ export function openShell(fullPath: string, shell?: string) {
     return spawn('START', [shell || 'cmd'], { shell: true, cwd: fullPath })
   }
 
+  if (__LINUX__) {
+    const commandArgs = ['--working-directory', fullPath]
+    return spawn('gnome-terminal', commandArgs, { shell: true })
```

Comment:
Is there a better way of figuring out the terminal program to use? Not all users will have `gnome-terminal` installed.

Reply:
I don't know its current status, but there is a script for launching the default terminal emulator:

- [xdg-terminal](https://github.com/Lazy-Newb-Pack/Lazy-Newb-Pack-Linux/blob/master/pack/xdg-terminal)
- [xdg-terminal: launching the preferred terminal emulator](https://lists.freedesktop.org/archives/xdg/2006-October/007058.html)
- [Can't run Dwarf Fortress](https://askubuntu.com/questions/631292/cant-run-dwarf-fortress)
- [A terminal emulator, xdg-terminal, is already in the xdg-utils CVS](http://airlied.livejournal.com/36651.html)

Reply:
@maidis would you be interested in opening up a PR to address this?